docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -258,5 +258,11 @@ There you can find links to upgrade notes for other versions too.
 ## [shopsys/coding-standards]
 - run `php phing standards-fix` to fix code style as we now disallow Yoda style for comparison in the Shopsys Framework coding standards ([#1209](https://github.com/shopsys/shopsys/pull/1209))
 
+## [shopsys/backend-api]
```

Comment:
That change isn't trivial - it is involved in multiple files like documentation, maybe configs, maybe tests... It would take I guess 2 hours of devel time to "do it"... Please come with this topic on standup

Reply:
We decided to solve this in separate PR. Resolving conversation here.